examples/lambda-layer/index.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright 2016-2017, Pulumi Corporation.  All rights reserved.
+
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+import { asset } from "@pulumi/pulumi";
+
+let lambda = new aws.lambda.LayerVersion("my-layer-version", {
+    code: new asset.AssetArchive({
+        "index.js": new asset.StringAsset(
+            "exports.handler = (e, c, cb) => cb(null, {statusCode: 200, body: 'Hello, world!'});",
+        ),
+    }),
+    compatibleRuntimes: [
+        aws.lambda.NodeJS8d10Runtime,
+        aws.lambda.NodeJS8d10Runtime,
+    ],
+    layerName: "lambda_layer_name",
+});
```

Comment:
Is the goal of the test to create a lambda function using this layer?  Or just to create the Layer resource?

Reply:
The goal of the test is just to make sure that we can create the layer based on the new parameters - I have added some outputs to verify that the layer has been created as expected